pkgs/top-level/python-packages.nix

```
@@ -1426,6 +1426,8 @@ in {
 
   credstash = callPackage ../development/python-modules/credstash { };
 
+  criticality_score = callPackage ../development/python-modules/criticality_score { };
```

```suggestion
  criticality-score = callPackage ../development/python-modules/criticality-score { };
```

Python packages need to follow the pep that converts _ to -.

---

Thanks for the review. Updated PR.